specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/sshPublicKey.json

```diff
@@ -371,6 +371,15 @@
             "type": "string",
             "description": "The name of the SSH public key."
           },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
```

Comment:
Just want to confirm, this is field is required? 
that makes it a breaking change.  

Shouldnt it be optional?  sshPublicKeys were being created fine before without this field

Reply:
yes, optional. let me fix